test/Conversion/ExportVerilog/hw-dialect.mlir

```diff
@@ -148,7 +148,6 @@ hw.module @TESTSIMPLE(%a: i4, %b: i4, %c: i2, %cond: i1,
 // CHECK-NEXT:   output                                                    r42);
 // CHECK:        wire [8:0][3:0] [[WIRE0:.+]] = {{[{}][{}]}}4'hF}, {4'hF}, {4'hF}, {4'hF}, {4'hF}, {4'hF}, {4'hF}, {4'hF}, {4'hF}};
 // CHECK-NEXT:   wire [2:0][3:0] [[WIRE1:.+]] = {{[{}][{}]}}4'hF}, {a + b}, {4'hF}};
-// CHECK-NEXT:   wire [9:0][3:0] [[WIRE2:array2d_idx_0_name]] = array2d[a];
```

Comment:
Essentially unrelated to the PR, we are testing a namehint with `array2d_idx_0_name` so can you move the namehint test to some other op?

Reply:
Will do - wasn't quite sure what that was testing.